OWNERS

```diff
@@ -6,3 +6,9 @@ approvers:
 - autoscaling-wg-leads
 - networking-wg-leads
 - knative-release-leads
+
+dir_denylist:
```

Comment:
Does this need to be [`owners_dir_denylist`](https://github.com/kubernetes/test-infra/blob/master/prow/config/config.go#L164)? (I know very little about Prow, so this could a silly question...)

Reply:
According to https://github.com/kubernetes/test-infra/blob/master/prow/ANNOUNCEMENTS.md `April 1st, 2021 The owners_dir_blacklist field in prow config has been deprecated in favor of owners_dir_denylist. The support of owners_dir_blacklist will be stopped in October 2021.`, but there might be an option for just `dir_denylist` :shrug: